Classes/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -131,6 +131,55 @@ public function nodeType(string $nodeType): QueryBuilderInterface
         return $this->queryFilter('term', ['neos_type_and_supertypes' => $nodeType]);
     }
 
+    /**
+     * Filter multiple node types
+     *
+     * @param array $expectedNodeTypes NodeTypes that should be expected
+     * @param array $excludedNodeTypes NodeTypes that should be excluded
+     * @return ElasticSearchQueryBuilder
+     * @throws QueryBuildingException
+     * @api
+     */
+    public function nodeTypeFilter(array $expectedNodeTypes, array $excludedNodeTypes): QueryBuilderInterface
+    {
+        $excludeShould = [];
```

Comment:
Nitpick: It ends in an bool must / must_not query so should is a bit missleading here

Reply:
I changed `Should` to `Terms`